### PR TITLE
atlasaction: add code suggestion comments to migrate lint action

### PR DESCRIPTION
**Pull request by ronenlu**
[see](https://github.com/ariga/atlas-action/pull/123#discussion_r1494486323):

image

### CODE REVIEW

File: atlasaction/action.go [hunk 1]
```diff
@@ -133,17 +134,19 @@ func MigrateLint(ctx context.Context, client *atlasexec.Client, act *githubactio
...
 		act.SetOutput("report-url", payload.URL)
 	}
-	dirName := act.GetInput("dir-name")
 	var summary bytes.Buffer
 	if err := comment.Execute(&summary, &payload); err != nil {
 		return err
 	}
-	if err := publish(act, dirName, summary.String()); err != nil {
+	if err := addSummary(act, summary.String()); err != nil {
 		return err
```
Comment (ronenlu) [inline atlasaction/action.go:142]: Testing the API

Comment (ronenlu) [inline atlasaction/action.go:142]:
```suggestion
	do other stuff instead
```

Comment (ronenlu) [inline atlasaction/action.go:142]:
```suggestion
	do other stuff instead
```

Comment (ronenlu) [inline atlasaction/action.go:142]:
```suggestion
	do other stuff instead
```

Comment (ronenlu) [inline atlasaction/action.go:142]:
```suggestion
	do other stuff instead
```

Comment (ronenlu) [inline atlasaction/action.go:142]:
```suggestion
	do other stuff instead
```

File: atlasaction/action.go
Comment (ronenlu) [inline atlasaction/action.go:1]:
```suggestion
	do other stuff instead
```

File: atlasaction/action_test.go [hunk 2]
```diff
@@ -355,20 +355,20 @@ func TestMigrateLint(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, 0, len(out))
 	})
-	t.Run("lint with error", func(t *testing.T) {
```
Comment (ronenlu) [inline atlasaction/action_test.go:358]: this test is included in the test case bellow: `lint summary - lint error`, thus I deleted it

**Review by a8m — APPROVED** [review]

**Pull request merged**